docs/ref/contrib/gis/install/index.txt

```diff
@@ -336,8 +336,8 @@ __ http://pdb.finkproject.org/pdb/browse.php?summary=django-gis
 MacPorts
 ~~~~~~~~
 
-`MacPorts`__ may be used to install GeoDjango prerequisites on Macintosh
-computers running OS X.  Because MacPorts still builds the software from source,
+`MacPorts`__ may be used to install GeoDjango prerequisites on Apple
```

Comment:
I guess "Apple" could be chopped since it could be possible to run macOS on non-Apple hardware.